pkg/reconciler/instances/istio/README.md

```diff
@@ -30,3 +30,33 @@ Follow these steps to run Istio Reconciler on your local machine:
    ```bash
     ./bin/mothership-darwin local --value global.ingress.domainName=example.com,global.domainName=example.com --components cluster-essentials,istio
    ```
+
+## Details
+
+Reconciliation in Kyma is handled by Reconciler. The Mothership Reconciler knows the reconciliation status of every managed Kyma cluster and initiates reconciliation of all Kyma components.
+
+The reconciliation of Istio is split into two different Kyma components: Istio and Istio Resources. The separation enables having the pure installation process of Istio under control via Istio Reconciler implementation and the Reconciler framework. The Reconciler framework functionality simply applies charts to the cluster for Istio Resources.
+
+The Istio component installs Istio on a Kyma cluster. For installation purposes, it requires Reconciler that uses `istioctl` and a rendered `istio-operator.yaml` file with Kyma-specific configurations. After proper installation/upgrade of the Istio service mesh within the cluster, the Istio proxies are restarted if needed. Within the whole Kyma reconciliation process, the installation of Istio is a prerequisite as other components rely on the proper service mesh setup and on the installed CRDs.
```

Comment:
```suggestion
The **Istio** component installs Istio on a Kyma cluster. The installation needs a Reconciler that uses `istioctl` and a rendered `istio-operator.yaml` file with Kyma-specific configurations. After installation (or upgrade) of the Istio service mesh within the cluster, the Istio proxies are restarted if needed. Within the whole Kyma reconciliation process, the installation of Istio is a prerequisite because other components rely on the service mesh setup and on the installed CRDs.
```
not sure we need "proper" here.